public/app/plugins/datasource/elasticsearch/types.ts

```diff
@@ -1,13 +1,152 @@
+export interface PipelineVariable extends Schema.PipelineVariable {}
+export interface MetricAggregationWithField extends Schema.MetricAggregationWithField {}
+export interface MetricAggregationWithMissingSupport extends Schema.MetricAggregationWithMissingSupport {}
+export type InlineScript = Schema.InlineScript;
+export interface MetricAggregationWithInlineScript extends Schema.MetricAggregationWithInlineScript {}
+export interface Count extends Schema.Count {}
+export interface Average extends Schema.Average {}
+export interface Sum extends Schema.Sum {}
+export interface Max extends Schema.Max {}
+export interface Min extends Schema.Min {}
+export type ExtendedStatMetaType = Schema.ExtendedStatMetaType;
+export interface ExtendedStat extends Schema.ExtendedStat {}
+export interface ExtendedStats extends Schema.ExtendedStats {}
+export type MetricAggregationWithMeta = Schema.ExtendedStats;
+export interface Percentiles extends Schema.Percentiles {}
+export interface UniqueCount extends Schema.UniqueCount {}
+export interface RawDocument extends Schema.RawDocument {}
+export interface RawData extends Schema.RawData {}
+export interface Logs extends Schema.Logs {}
+export interface Rate extends Schema.Rate {}
+// export interface BasePipelineMetricAggregation extends Schema.BasePipelineMetricAggregation{}
+// export interface PipelineMetricAggregationWithMultipleBucketPaths extends Schema.PipelineMetricAggregationWithMultipleBucketPaths{}
+// export type MovingAverageModelSettings = Schema.MovingAverageModelSettings
+// export interface MovingAverage extends Schema.MovingAverage{}
+export type MovingAverageModel = Schema.MovingAverageModel;
+export interface MovingAverageModelOption extends Schema.MovingAverageModelOption {}
+export interface BaseMovingAverageModelSettings extends Schema.BaseMovingAverageModelSettings {}
+export interface MovingAverageSimpleModelSettings extends Schema.MovingAverageSimpleModelSettings {}
+export interface MovingAverageLinearModelSettings extends Schema.MovingAverageLinearModelSettings {}
+export interface MovingAverageEWMAModelSettings extends Schema.MovingAverageEWMAModelSettings {}
+export interface MovingAverageHoltModelSettings extends Schema.MovingAverageHoltModelSettings {}
+export interface MovingAverageHoltWintersModelSettings extends Schema.MovingAverageHoltWintersModelSettings {}
+export interface MovingFunction extends Schema.MovingFunction {}
+export interface Derivative extends Schema.Derivative {}
+export interface SerialDiff extends Schema.SerialDiff {}
+export interface CumulativeSum extends Schema.CumulativeSum {}
+export interface BucketScript extends Schema.BucketScript {}
+export interface TopMetrics extends Schema.TopMetrics {}
+export type PipelineMetricAggregation = Schema.PipelineMetricAggregation;
+export type MetricAggregationWithSettings = Schema.MetricAggregationWithSettings;
+export interface ElasticsearchQuery extends Schema.Elasticsearch {}
```

Comment:
```suggestion
import {
  BucketAggregationType,
  Filter,
  MetricAggregation,
  MetricAggregationType,
  MovingAverageEWMAModelSettings,
  MovingAverageHoltModelSettings,
  MovingAverageHoltWintersModelSettings,
  MovingAverageLinearModelSettings,
  MovingAverageModel,
  MovingAverageSimpleModelSettings,
  PipelineMetricAggregationType,
  TermsOrder,
} from './dataquery.gen';

export * from './dataquery.gen';
export { Elasticsearch as ElasticsearchQuery } from './dataquery.gen';
```

I think you can do this instead, the explicitly defined types will override the `*` import and the rest won't need to be declared explicitly.

Reply:
Updated in https://github.com/grafana/grafana/pull/62147/commits/af4aaf2f37f69884a66b3098dc262c9d9734e1d0